LiskHQ/lisk-desktop
# Disable send button if no balance

### What was the problem?

This PR resolves #5215

### How was it solved?

Check current user for token balances to determine if send button should be disabled

### How was it tested?

1. Select an account with no balance Expected: Send button should be disabled.
2. Search for an account with balance, for example: [/explore](http://0.0.0.0:8080/#/explorer?address=lskguo9kqnea2zsfo3a6qppozsxsg92nuuma3p7ad)
3. Expected: Send button should be disabled.
